Token_Contracts/contracts/HumanStandardTokenFactory.sol

```diff
@@ -54,10 +54,9 @@ contract HumanStandardTokenFactory {
 
     function createHumanStandardToken(uint256 _initialAmount, string _name, uint8 _decimals, string _symbol) returns (address) {
 
-        address newTokenAddr = address(new HumanStandardToken(_initialAmount, _name, _decimals, _symbol));
-        HumanStandardToken newToken = HumanStandardToken(newTokenAddr);
+        HumanStandardToken newToken = (new HumanStandardToken(_initialAmount, _name, _decimals, _symbol));
         newToken.transfer(msg.sender, _initialAmount); //the factory will own the created tokens. You must transfer them.
-        created[msg.sender].push(newTokenAddr);
-        return newTokenAddr;
+        created[msg.sender].push(address(newToken));
+        return address(newToken);
```

Comment:
One less line of code, and more intuitive. First create contract, then use the address of the contract.